src/components/Admin/Admin.test.jsx

```diff
@@ -294,4 +295,222 @@ describe('<Admin />', () => {
       expect(mockGetDashboardAnalytics).toBeCalled();
     });
   });
+
+  describe('calls download csv fetch method for table', () => {
+    let spy;
+
+    afterEach(() => {
+      spy.mockRestore();
+    });
+
+    it('enrollments', () => {
```

Comment:
You could consider looping over fetchUrl/actionSlug pairs to avoid a lot of the duplication here. Might make it easier to read since theres a lot of boilerplate creating the AdminWrapper